src/Workspaces/MSBuildTest/MSBuildWorkspaceTests.cs

```diff
@@ -94,7 +94,9 @@ public async Task TestOpenSolution_MultiProjectSolution()
             }
         }
 
-        [ConditionalFact(typeof(VisualStudioMSBuildInstalled)), Trait(Traits.Feature, Traits.Features.MSBuildWorkspace)]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/41456")]
+        // When this test is unskipped, the following should be used instead of the Fact attribute.
+        // [ConditionalFact(typeof(VisualStudioMSBuildInstalled)), Trait(Traits.Feature, Traits.Features.MSBuildWorkspace)]
```

Comment:
>// [ConditionalFact(typeof(VisualStudioMSBuildInstalled)), Trait(Traits.Feature, Traits.Features.MSBuildWorkspace)] [](start = 8, length = 115)

how does this change relate to the present PR?

Reply:
I want to be able to run tests without this failing every time.

Reply:
I didn't follow. Did the changes in this PR cause this problem? Was this failing prior to your change?

Reply:
Yes, this was failing consistently on my home machine prior to my change, even in master.